decred/decred/wallet/wallet.py

```diff
@@ -87,13 +70,11 @@ def initialize(self, seed, pw):
         self.masterDB[DBKeys.checkKey] = pwKey.encrypt(CHECKPHRASE)
         self.masterDB[DBKeys.keyParams] = crypto.ByteArray(pwKey.params().serialize())
         db = self.coinDB.child(str(BipIDs.decred), table=False)
-        acctManager = accounts.createNewAccountManager(
-            root, cryptoKey, "dcr", cfg.net, db
-        )
+        acctManager = accounts.createNewAccountManager(root, cryptoKey, "dcr", net, db)
         self.coinDB[BipIDs.decred] = acctManager
 
     @staticmethod
-    def create(path, password):
+    def create(path, password, net):
         """
         Create a wallet, locked by `password`. The seed will be randomly
         generated.
```

Comment:
The `chain` arg is actually `net` (ideally `netParams`). The returned values are described in opposite order.